packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -1100,6 +1100,7 @@ class Dropdown extends AutoControlledComponent<WithAsProp<DropdownProps>, Dropdo
     const { disabled } = this.props;
 
     const handleInputKeyDown = (e: React.SyntheticEvent, searchInputProps: DropdownSearchInputProps) => {
+      e.stopPropagation();
```

Comment:
merge conflict?

Reply:
I was waiting you to have a look in the type itself so then I would solve the merge conflicts.